app/views/transactions/shared/_transaction.html.erb

```diff
@@ -0,0 +1,7 @@
+<li class="transaction-item mb-2">
+    <h3><%=transaction.name%>: <span class="transaction-price">$<%=transaction.amount%></span></h3>
+    <em>
+        <i class="bi bi-calendar"></i>
+        13 Apr 2014 at 3:00 pm
+    </em>
```

Comment:
- [ ] The transaction date should be the date when it was created and not the hard-coded value.